activerecord/test/cases/locking_test.rb

```diff
@@ -177,6 +177,16 @@ def test_touch_existing_lock
     assert_equal 1, p1.lock_version
   end
 
+  def test_touch_stale_object
+    p1 = Person.create!(first_name: 'Mehmet Emin')
+    p2 = Person.find(p1.id)
+    p1.update_attribute(:gender, 'M')
+
+    assert_raises(ActiveRecord::StaleObjectError) do
+      p2.touch
```

Comment:
Let's go for a bit more readability and change p1 to person and p2 to stale_person. That way we communicate the coming expectation throughout the test.